src/main/java/io/vertx/ext/mongo/ValidationOptions.java

```diff
@@ -10,12 +10,12 @@
 
 @DataObject(generateConverter = true)
 public final class ValidationOptions {
-  private JsonObject validator = new JsonObject();
+  private JsonObject validator;
```

Comment:
This is a breaking change.

If you want to avoid configuring the Mongo driver object, you could check if the Json object is empty.

Reply:
I thought it's cleaner to not instantiate a JsonObject if it's probably not needed anyway. Also it kind of makes it easier to uniformly assert it when passing it to the driver without having to be type aware. Anyhow, will change it as you wish.

Reply:
It's not a bad idea but we try to limit the changes to the minimum. It's always painful for users to get breaking changes in a micro release.